StreamChatClient_v3/APIClient/Endpoints/ChannelEndpoints.swift

```diff
@@ -62,12 +62,10 @@ struct ChannelEndpointPayload<ExtraData: ExtraDataTypes>: Decodable {
 struct ChannelDetailPayload<ExtraData: ExtraDataTypes>: Decodable {
     let cid: String
```

Comment:
Why you don't want to use `ChannelId` here as self decodable type and use `CustomStringConvertible` for DTO?

Reply:
Hmmm, that's actually a better solution 👍 The decoding will fail if the channel id has an invalid format. Thanks

Reply:
fixed in e6eb0616e1d24b910320d74cbc0591efc2d4000e